src/main/java/org/jenkinsci/plugins/envinject/service/BuildCauseRetriever.java

```diff
@@ -46,6 +48,28 @@
         return env;
     }
 
+    public String getCauseUserName(AbstractBuild<?, ?> build) {
```

Comment:
The method needs to be annotated an documented  properly.
Also the parameter should be <code>Run<?,?></code> unless you need something from the Abstract build